.templates/zigbee2mqtt/service.yml

```diff
@@ -1,10 +1,14 @@
   zigbee2mqtt:
     container_name: zigbee2mqtt
-    image: koenkk/zigbee2mqtt
+    build: ./.templates/zigbee2mqtt/.
+    environment:
+      - TZ=Etc/UTC
+    ports:
+      - "8080:8080"
```

Comment:
Just as an FYI, 8080 is a commonly used port. There maybe conflicts using this port with some setups.